mne/preprocessing/realign.py

```diff
@@ -62,7 +63,11 @@ def realign_raw(raw, other, t_raw, t_other, verbose=None):
         warn('Fewer than 20 times passed, results may be unreliable')
 
     # 1. Compute correction factors
-    coef = np.polyfit(t_other, t_raw, deg=1)
+    poly = Polynomial.fit(x=t_other, y=t_raw, deg=1)
+    converted = poly.convert(domain=(-1, 1))
+    [zero_ord, first_ord] = converted.coef
```

Comment:
just a sanity check here, because I couldn't easily tell from the numpy docs:  is this order (zero_ord, first_ord) correct?  I notice below you're replacing the old `coef[0]` with `first_ord` and the old `coef[1]` with `zero_ord`.

Reply:
never mind, I found the answer in the transition guide.  before I was looking only at the docs of the new class, which don't tell you the coef order.